prometheus-metrics/src/main/scala/org/http4s/metrics/prometheus/PrometheusMetricsSettings.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.metrics.prometheus
+
+import cats.data.NonEmptyList
+
+final case class PrometheusMetricsSettings(
+    metricsNames: PrometheusMetricsNames,
+    responseDurationSecondsHistogramBuckets: NonEmptyList[Double]
+) {
+  def withMetricsNames(metricsNames: PrometheusMetricsNames): PrometheusMetricsSettings =
+    copy(metricsNames = metricsNames)
+
+  def withResponseDurationSecondsHistogramBuckets(
+      responseDurationSecondsHistogramBuckets: NonEmptyList[Double]
+  ): PrometheusMetricsSettings =
+    copy(responseDurationSecondsHistogramBuckets = responseDurationSecondsHistogramBuckets)
+}
+
+object PrometheusMetricsSettings {
+  // https://github.com/prometheus/client_java/blob/parent-0.6.0/simpleclient/src/main/java/io/prometheus/client/Histogram.java#L73
+  private val DefaultHistogramBuckets: NonEmptyList[Double] =
```

Comment:
It _might_ be useful to have this public. A common use case is merely adding more buckets, or scaling all buckets by a constant scalar.

If we made it public, I'd also make this a `NonEmptySet`.

Reply:
It could be used via `PrometheusMetricsSettings.DefaultSettings`, but yeah, it makes sense to have it public.